frontend/src/container/GridCardLayout/DashboardEmptyState/DashboardEmptyState.styles.scss

```diff
@@ -0,0 +1,244 @@
+.dashboard-empty-state {
+	display: flex;
+	flex-direction: column;
+	align-items: center;
+	justify-content: center;
+	height: 500px;
+
+	.dashboard-content {
+		display: flex;
+		flex-direction: column;
+		align-items: flex-start;
+		gap: 12px;
+
+		.heading {
+			display: flex;
+			flex-direction: column;
+			gap: 4px;
+
+			.icons {
+				color: white;
+			}
+
+			.welcome {
+				color: var(--Vanilla-400, #c0c1c3);
```

Comment:
better to use - `var(--bg-vanilla-400)` format

Reply:
might have missed this , updating

Reply:
done